Src/Base/AMReX_GpuLaunchFunctsC.H

```diff
@@ -13,15 +37,25 @@ namespace detail {
     auto call_f_scalar_handler (F const& f, N i)
         noexcept -> decltype(f(0))
     {
-        f(i);
+        f(SIMDindex<1, N>{i});
```

Comment:
This would add the simd index type to code using regular ParallelFor. Since there is only one way to call the user function in ParallelForSIMD (not with and without Gpu::Handler like the other ParallelFor), it would be best to not use the call_f funcitons and just do it directly in ParallelForSIMD.